tests/test_parser.py

```diff
@@ -781,6 +781,82 @@ def function() -> int:
         assert parser.raw_statements == {1, 3, 4, 5, 6, 8, 9}
         assert parser.statements == {1, 8, 9}
 
+    def test_multiline_exclusion_single_line(self) -> None:
+        regex = r"print\('.*'\)"
+        parser = self.parse_text("""\
+        def foo():
+            print('Hello, world!')
+        """, regex)
```

Comment:
Please indent the code strings like this:
```suggestion
        parser = self.parse_text("""\
            def foo():
                print('Hello, world!')
            """, regex)
```
This should match the way other code is included in tests.

Reply:
Thank you! Done.